ext_localconf.php

```diff
@@ -1,20 +1,22 @@
 <?php
+
+use Causal\Oidc\Controller\AuthenticationController;
+use Causal\Oidc\Service\AuthenticationService;
+use TYPO3\CMS\Core\Configuration\ExtensionConfiguration;
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Utility\ExtensionUtility;
+
 defined('TYPO3') or die();
 
 (static function (string $_EXTKEY) {
-    if (class_exists(\TYPO3\CMS\Core\Authentication\AuthenticationService::class)
-        && !class_exists(\TYPO3\CMS\Sv\AuthenticationService::class)) {
-        class_alias(\TYPO3\CMS\Core\Authentication\AuthenticationService::class, \TYPO3\CMS\Sv\AuthenticationService::class);
-    }
-
-    // Configuration of authentication service
-    // TODO: Use proper TYPO3 API
-    $settings = $GLOBALS['TYPO3_CONF_VARS']['EXTENSIONS'][$_EXTKEY] ?? [];
+    $extConfig = GeneralUtility::makeInstance(ExtensionConfiguration::class);
```

Comment:
nitpicking but I don't see the benefit of `$extConfig`, I prefer using `->get` directly as done in all other places.

Reply:
True. Will change that